apps/storefront/docs/patterns/grid-alignment/grid-alignment.mdx

```diff
@@ -5,3 +5,29 @@ mode: draft
 route: /patterns/grid-alignment/
 ---
 
+**The organization of space is key to every great design.  By aligning your component to a grid, you will provide visual structure throughout your product.**
+
+## Grid Reminder
+
+- It is the responsibility of the designer to define a maximum width and choose the correct grid.
+- It is up to the designer to make sure the design will scale correctly on all sizes. Start with a viewport that is the most common for the users. Make sure to test that the content will fit at smaller and larger viewports.
+- It is the responsibility of the developers to implement the maximum width and use the grid as designed.
+
+## Guidelines
+
+Using a grid as a starting point for your component implementation will help you arrange your content into structured propositions. You also have to keep in mind that EDS is based on an 8px ratio. This means that you always have to keep a minimum of 8px beetween objects and elements.
+
+### Don’t
+
+- Do not use less than 8px between each components. 
+- If some of your objects are aligned to the grid do not set other objects off-grid.
+
+### Align object within a frame
+
+As you have applied the right grid size to your frame, you can use the columns to place your objects. 
+It is recommended that your keep all your objects within the columns. 
+In some cases you might need to be off-grid. You should then keep in mind the 8px ratio beetween elements
```

Comment:
Small mistakes, remove extra letters and right grid should be replaced with correct since that is the proper word to use and doesn't confuse with left/right 

> As you have applied the ~~right~~grid size to your frame, you can use the columns to place your objects. 
> It is recommended that you~~r~~ keep all your objects within the columns. 
> In some cases you might need to be off-grid. You should then keep in mind the 8px ratio be~~e~~tween elements

Reply:
Updated